packages/integrations/markdoc/src/index.ts

```diff
@@ -233,12 +233,11 @@ export const Content = createComponent({
 			},
 			'astro:server:setup': async ({ server }) => {
 				server.watcher.on('all', (event, entry) => {
-					if (prependForwardSlash(pathToFileURL(entry).pathname) === markdocConfigResultId) {
-						console.log(
-							yellow(
-								`${bold('[Markdoc]')} Restart the dev server for config changes to take effect.`
-							)
-						);
+					for (const markdocConfigBase of SUPPORTED_MARKDOC_CONFIG_FILES) {
+						const absolutePath = new URL(markdocConfigBase, astroConfig.root);
+						if (entry === fileURLToPath(absolutePath)) {
```

Comment:
I assume this _will_ be the file path and not a Vite ID. Let me know if that's wrong!